machinery-helper/machinery_helper.go

```diff
@@ -236,11 +263,86 @@ func findUnmanagedFiles(dir string, rpmFiles map[string]string, rpmDirs map[stri
 	}
 }
 
+func amendMode(entry *UnmanagedFile, perm os.FileMode) {
+	if entry.Type == "link" {
+		return
+	}
+
+	result := int64(perm.Perm())
+
+	if perm&os.ModeSticky > 0 {
+		result |= 01000
+	}
+	if perm&os.ModeSetuid > 0 {
+		result |= 04000
+	}
+	if perm&os.ModeSetgid > 0 {
+		result |= 02000
+	}
+	entry.Mode = strconv.FormatInt(result, 8)
+}
+
+func dirInfo(path string) (size int64, fileCount int) {
+	files, _ := readDir(path)
+
+	size = int64(0)
+	fileCount = len(files)
+	for _, f := range files {
+		if f.IsDir() {
+			if _, ok := IgnoreList[path + f.Name()]; !ok {
+				subSize, subCount := dirInfo(path + f.Name() + "/")
+				size += subSize
+				fileCount += subCount
+			}
+		} else if f.Mode()&os.ModeSymlink != os.ModeSymlink {
```

Comment:
I prefer the way it is right now and go fmt doesn't have a preference for this and go sources like the os package use it without the spaces when doing a validation like in

```
func (m FileMode) IsRegular() bool {
	return m&ModeType == 0
}
```
https://golang.org/src/os/types.go?s=3244:3278#L83

They also seem to use it the other way around when just doing the bit operation like in https://golang.org/src/os/types.go?s=3355:3388#L88

We could start defining this kind of thing on a